### DISCUSSION TITLE
Only Show HasMany

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#2093] [Help]
Is it possible in a hasmany relation, to hide the main panel after saving the record and show only the tabs to load the hasmany relation records?

**Comment by danharrin** [comment]
Sorry, not sure what you mean

**Comment by Z3d0X** [comment]
on save redirect to view page

In view Page you could hide the form & show the relation manager?

**Reply by jcgmalave** [reply]
exactly this is what i mean: In view Page you could hide the form & show the relation manager

**Reply by danharrin** [reply]
You could define a custom `$view` for that page and remove the form code from the original view?
